README.md

```diff
@@ -52,3 +52,66 @@ The script which is responsible for streaming the raw CAN data into the database
 Currently to use the script run main and when prompted input the path to the .mf4 file you want to have uploaded.<br>
 
 As we do not have means to collect context data yet, we use a dummy context with an id of 1. This is gross and bad. We will be removing that as soon as we have a way to query the user for context data.<br>
+
+## Web UI 
+
+This react-strap based website is responsible for gathering all the context relating to the data we gather.<br>
```

Comment:
This is going to be a single page of many in this website which collects more than just the context.